packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -96,6 +129,15 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
     }
 
     protected readonly overrideIdentifiers = new Set<string>();
+
+    /**
+     * Register a new overrideIdentifier. Existing identifiers are not replaced.
+     *
+     * Allows overriding existing values while keeping both values in store
```

Comment:
```suggestion
     * Allows overriding existing values while keeping both values in store.
```

Reply:
I think either we add the punctuation, or the following line should not be capitalized to:

```
* Allows overriding existing values while keeping both values in store, 
* for example to store different editor settings, e.g. "[markdown].editor.autoIndent",
```